### PR TITLE
[vmm]: fix initrd building error in aarch64 enviroment

### PR DESCRIPTION
**Pull request by flyflypeng**
fix the issue #25 
1. fix the build rootfs container left problem when build-script return error
2. install golang package directly from [golang](https://go.dev/dl/)
3. format the shell scripts by `shfmt` tool.

### CODE REVIEW

**Review by Vanient — APPROVED** [review]

File: vmm/scripts/image/centos/build_rootfs.sh [hunk 1]
```diff
@@ -15,40 +15,50 @@
...
 }
 
+install_golang() {
+    pushd /home/
+    rm -f go${golang_version}.linux-arm64.tar.gz
+    wget "https://go.dev/dl/go${golang_version}.linux-arm64.tar.gz"
+    rm -rf /usr/local/go && tar -C /usr/local -xzf "go${golang_version}.linux-arm64.tar.gz"
+    echo "export PATH=$PATH:/usr/local/go/bin" >>/etc/profile
+    source /etc/profile
+    go version
```
Comment (Burning1020) [inline vmm/scripts/image/centos/build_rootfs.sh:56]: need execute permission

**Review by Burning1020 — CHANGES_REQUESTED** [review]

File: vmm/scripts/image/centos/build_rootfs.sh [hunk 2]
```diff
@@ -15,40 +15,50 @@
...
     source vmm/scripts/image/install_rust.sh
 
     make bin/vmm-task ARCH=${ARCH}
     popd
 }
 
+install_golang() {
+    pushd /home/
+    rm -f go${golang_version}.linux-arm64.tar.gz
+    wget "https://go.dev/dl/go${golang_version}.linux-arm64.tar.gz"
```
Comment (Burning1020) [inline vmm/scripts/image/centos/build_rootfs.sh:52]: please download the arch based tar file

**Review by Burning1020 — CHANGES_REQUESTED** [review]

File: vmm/scripts/image/centos/build_rootfs.sh (hunk 2 above)
Comment (flyflypeng) [reply vmm/scripts/image/centos/build_rootfs.sh]: Fixed！

File: vmm/scripts/image/centos/build_rootfs.sh (hunk 1 above)
Comment (flyflypeng) [reply vmm/scripts/image/centos/build_rootfs.sh:72]: The `go` binary extracts from the go${golang_version}.linux-${arch}.tar.gz keeps the execution permission as default, so we don't need to change it.

**Review by Burning1020 — APPROVED** [review]
LGTM

**Pull request merged**